core/dbt/adapters/internal_storage/local_filesystem.py

```diff
@@ -64,7 +64,7 @@ def read(
 def write(
     path: str,
     content: Union[str, bytes, None],
-    overwrite: bool = False,
+    overwrite: bool = True,
```

Comment:
Maybe add a comment as to why we chose to default to true for the future

Reply:
Do we actually have any places where we don't want to overwrite the existing file?

Reply:
@leahwicz TBH, I didn't choose the default-- this function reflects the way the code was written before SAs.  The change here just reflects that I didn't get it right the first time (which @gerda pointed out)